src/borg/testsuite/xattr.py

```diff
@@ -1,81 +1,84 @@
 import os
-import tempfile
-import unittest
 
 import pytest
 
 from ..platform.xattr import buffer, split_lstring
 from ..xattr import is_enabled, getxattr, setxattr, listxattr
 from ..platformflags import is_linux
-from . import BaseTestCase
 
 
-@unittest.skipUnless(is_enabled(), "xattr not enabled on filesystem")
-class XattrTestCase(BaseTestCase):
-    def setUp(self):
-        self.tmpfile = tempfile.NamedTemporaryFile()
-        self.symlink = self.tmpfile.name + ".symlink"
-        os.symlink(self.tmpfile.name, self.symlink)
+@pytest.mark.skipif(not is_enabled(), reason="xattr is not enabled on filesystem")
+@pytest.fixture()
+def tempfile_symlink(tmp_path):
+    temp_file = open(os.fspath(tmp_path / "xattr"), "w")
```

Comment:
I _think_ this is the equivalent using tmp_path without changing the tests too much.

Reply:
guess it would be good to use
```
with open(...) as temp_file:
    ...
    yield temp_file, symlink
    # i don't think we need to unlink the symlink here as pytest
    # should erase everything in tmp_path anyway after the test
    # is finished, but i do think closing temp_file is useful as some
    # platforms might have troubles if we leave files open.

Reply:
btw, I also had a look at `is_enabled`:

it has has a path param to give the path that should be checked for xattr capability (default is the default temp path used by python)

guess it depends on how exactly tmp_path works whether that works correctly now, but we could improve by giving the correct path as a param.

Reply:
> with open(...) as temp_file:

Really good idea to use a CM here. Just implemented this as you suggested.

> we could improve by giving the correct path as a param

I agree. Done 👍